bin/maintenance

```diff
@@ -0,0 +1,86 @@
+#! /bin/bash
+
+maintenance=false
+environment=prod
```

Comment:
I would rather not have any defaults and have the script error out if either of these parameters are not provided.

Reply:
Done